src/infrastructure.sh

```diff
@@ -1292,6 +1292,44 @@ rlServiceRestore() {
     return $failed
 }
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# rlServiceRestore
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+: <<'=cut'
+=pod
+
+=head3 rlServiceStatus
+
+Print status of given C<service>.
+
+    rlServiceStatus service [service...]
+
+=over
+
+=item service
+
+Name of the service(s) to get the state.
+
+=back
+
+Returns status the last service.
+
+=cut
+
+rlServiceStatus() {
+    local IFS
+    local res
+    res=0
+
```

Comment:
Just a nitpick: this newline seems extraneous...

Reply:
yes, it is not necessary, just a visual separator